marvin/report/observers/event_logger.py

```diff
@@ -47,6 +92,23 @@ def on_step_ended(self, event):
         self._p("%s[%s] %s (%d ms)", self._indent * step.level, status, step.name, event.duration)
 
     def on_test_started(self, event):
+        # Check if the test's title is defined on the setup for the summary
+        # report. If not, we use the name of test script
+        test_name = event.test_script.name
+        setup_data = event.data_provider.setup_data()
+        if "test_report_title" in setup_data \
+                and setup_data["test_report_title"] != "":
+            test_name = setup_data["test_report_title"]
```

Comment:
I think this is good for now. But I would prefer (it can be in a future PR) if we can add an optional metadata section in data files instead of piggy backing on the setup test data. With the metadata think we could redefine the test title, description, tags, etc.